.env.example

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL=postgres://<user>:<pass>@localhost:5432/goodtables
+BROKER_URL=redis://localhost:6379/0
+RESULT_BACKEND=redis://localhost:6379/1
```

Comment:
It would be good to not need this anymore and just use postgres. For get_task we can query pg (we might need to store a pending status object when the task is finished